services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/models/snmp/OIDMap.java

```diff
@@ -0,0 +1,95 @@
+package us.dot.its.jpo.ode.api.models.snmp;
+
+import java.util.Map;
+
```

Comment:
Is this NTCIP-1218 based? It may be useful to have a comment that states that if we don't include it in the String key.